src/iidm/converter/xml/NetworkXml.cpp

```diff
@@ -27,8 +27,10 @@
 
 #include "xml/XmlEncoding.hpp"
 
+#include "AliasesXml.hpp"
 #include "HvdcLineXml.hpp"
 #include "LineXml.hpp"
+#include "PropertiesXml.hpp"
```

Comment:
It's strange you have to add it now. Fro which file does this include came from?

Reply:
I did not have to, it was probably undirectly included, I prefered to add it now even it is not relevant in this PR :/
It came from 
#include "HvdcLineXml.hpp"
#include "LineXml.hpp"

Both inherit from template class `AbstractIdentifiableXml` which needs it...

I revert ?